internal/archive/archive.go

```diff
@@ -147,6 +151,39 @@ func (a *ubuntuArchive) Info(pkg string) (*PackageInfo, error) {
 const ubuntuURL = "http://archive.ubuntu.com/ubuntu/"
 const ubuntuPortsURL = "http://ports.ubuntu.com/ubuntu-ports/"
 
+var proArchiveInfo = map[string]struct {
+	BaseURL, Label string
+}{
+	setup.ProFIPS: {
+		BaseURL: "https://esm.ubuntu.com/fips/ubuntu/",
+		Label:   "UbuntuFIPS",
+	},
+	setup.ProFIPSUpdates: {
+		BaseURL: "https://esm.ubuntu.com/fips-updates/ubuntu/",
+		Label:   "UbuntuFIPSUpdates",
+	},
+	setup.ProApps: {
+		BaseURL: "https://esm.ubuntu.com/apps/ubuntu/",
+		Label:   "UbuntuESMApps",
+	},
+	setup.ProInfra: {
+		BaseURL: "https://esm.ubuntu.com/infra/ubuntu/",
+		Label:   "UbuntuESM",
+	},
+}
+
+// archiveURL returns the archive base URL depending on the "pro" value and
+// selected architecture "arch".
```

Comment:
_... depending on the pro and arch parameters._

These are variables, so we call them by name in docs, not stringified.

With that said, this comment can be dropped completely, as it's literally saying that archiveURL returns archiveURL, based on the two parameters it takes, which is always the case for functions. :)

Reply:
Done in https://github.com/canonical/chisel/pull/167/commits/3d457a9f712677332fc7fec6d878962671d4ed3d.